db/rdbms/src/main/resources/mapper/SequenceFlowMapper.xml

```diff
@@ -52,6 +52,84 @@
             #{historyCleanupDate, jdbcType=TIMESTAMP})
   </insert>
 
+  <insert id="insertIfNotExists" parameterType="io.camunda.db.rdbms.write.domain.SequenceFlowDbModel" databaseId="postgresql">
```

Comment:
💭 I think, theoretically we could create reusable SQL snippets for these. I'd rather not over engineer it right now. If we need these anywhere else in the future, then with that issue, we could try to generalize them as well.

Reply:
I agree, let's see if we need it again and extract at that point. You would have to define multiple parts to this to hand in and then place appropriately, depending on the DB. Nah, not now.